games-strategy/dopewars/dopewars-1.5.12.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="Make a fortune dealing drugs on the streets of New York"
+DESCRIPTION="dopewars is a free Unix/Win32 rewrite of a game originally based \
+on \"Drug Wars\" by John E. Dell. The idea of dopewars is to deal in drugs on \
+the streets of New York, amassing a huge fortune and paying off the loan \
+shark, while avoiding the ever-annoying police. The Unix/Win32 rewrite, as \
+featuring a so-called \"antique\" mode which closely follows the original, \
+introduces new features such as the ability to take part in multi-player \
+games. dopewars aims to be highly configurable, and what you can't change in \
+the configuration files you can change by poking around in the source, which \
+is freely available under the terms of the GNU General Public License.
+
+ADVISORY: dopewars is not recommended for children below 16 without parental \
+supervision. We also do not condone real-life drug dealing."
+HOMEPAGE="https://dopewars.sourceforge.io/"
+COPYRIGHT="1999-2005 dopewars developers"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/dopewars/dopewars-$portVersion.tar.gz"
+CHECKSUM_SHA256="23059dcdea96c6072b148ee21d76237ef3535e5be90b3b2d8239d150feee0c19"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+GLOBAL_WRITABLE_FILES="
+	var/dopewars.sco keep-old
+	"
+
+PROVIDES="
+	dopewars = $portVersion
+	cmd:dopewars
+	"
+REQUIRES="
+	haiku
+	lib:libglib
+	lib:libncurses
+	lib:libSDL
+	lib:libSDL_mixer
```

Comment:
Could you update these with the correct naming? See https://github.com/haikuports/haikuports/issues/3878

Reply:
Oh, I *was* wondering what to name it since I appended `_2.0` for the SDL2 libs. Thanks!